runtime/doc/options.txt

```diff
@@ -5205,11 +5205,25 @@ A jump table for the options with a short description can be found at |Q_op|.
 	Note that such processing is done after |:set| did its own round of 
 	unescaping, so to keep yourself sane use |:let-&| like shown above.
 							*shell-powershell*
-	To use powershell: >
+	To use PowerShell: >
 		let &shell = has('win32') ? 'powershell' : 'pwsh'
-		set shellquote= shellpipe=\| shellxquote=
+		set shellquote= shellxquote=
 		set shellcmdflag=-NoLogo\ -NoProfile\ -ExecutionPolicy\ RemoteSigned\ -Command
-		set shellredir=\|\ Out-File\ -Encoding\ UTF8
+		set shellredir=2>&1\ \|\ Out-File\ -Encoding\ UTF8;\ exit\ $LastExitCode
+		set shellpipe=2>&1\ \|\ Out-File\ -Encoding\ UTF8;\ exit\ $LastExitCode
+
+< 	The above should work for all versions of PowerShell, but may result
+	in superfluous error object traces with PowerShell 5.1 due to how it
```

Comment:
Is powershell v5 what you have on your system? Windows 7 should be supported because of MSVC even if the OS is EOL. This means Powershell v2-5 should be considered if you're going to mention exact version numbers.

I suggest rewriting the paragraph to target Powershell v7.1 and suggest the above config for older powershell versions unless there's powershell v6 issue like with php.

Reply:
Rewrote for clarity; PTAL